openpifpaf/network/bottleneck_transformer.py

```diff
@@ -0,0 +1,255 @@
+import torch
+from torch import nn, einsum
+
+from einops import rearrange
+
+# translated from tensorflow code
+# https://gist.github.com/aravindsrinivas/56359b79f0ce4449bcb04ab4b56a57a2
```

Comment:
Both files have the MIT license, I have added the licensing to the docs folder and refer to those files, just like David did with his new backbones.
In case you also want to have a look:
EffNetV2 license: https://github.com/d-li14/efficientnetv2.pytorch/blob/main/LICENSE
BotNet license: https://github.com/lucidrains/bottleneck-transformer-pytorch/blob/main/LICENSE